azure/kubernetes/aks-kubernetes.md

```diff
@@ -0,0 +1,211 @@
+
+| Role       | Description     | Link     |
+| :------------- | :----------: | -----------: |
+|  Azure Kubernetes Service RBAC Viewer | Allows read-only access to see most objects in a namespace. It doesn't allow viewing roles or role bindings. This role doesn't allow viewing Secrets, since reading the contents of Secrets enables access to ServiceAccount credentials in the namespace, which would allow API access as any ServiceAccount in the namespace (a form of privilege escalation)   | https://docs.microsoft.com/en-us/azure/aks/concepts-identity#built-in-roles    |
+| Azure Kubernetes Service RBAC Writer   | Allows read/write access to most objects in a namespace. This role doesn't allow viewing or modifying roles or role bindings. However, this role allows accessing Secrets and running Pods as any ServiceAccount in the namespace, so it can be used to gain the API access levels of any ServiceAccount in the namespace. | https://docs.microsoft.com/en-us/azure/aks/concepts-identity#built-in-roles |
+| Azure Kubernetes Service RBAC Admin   | Allows admin access, intended to be granted within a namespace. Allows read/write access to most resources in a namespace (or cluster scope), including the ability to create roles and role bindings within the namespace. This role doesn't allow write access to resource quota or to the namespace itself. | https://docs.microsoft.com/en-us/azure/aks/concepts-identity#built-in-roles |
+| Azure Kubernetes Service RBAC Cluster Admin   |Allows super-user access to perform any action on any resource. It gives full control over every resource in the cluster and in all namespaces. | https://docs.microsoft.com/en-us/azure/aks/concepts-identity#built-in-roles |
+||||
+
+
+Secondly, there is the Kubernetes RBAC system.
+This permits fine-grained control of the Kubernetes resources themselves via Role and ClusterRole definitions.
+These are then linked with users via integration with Active Directory by RoleBindings and ClusterRoleBindings.
+
+There is a feature in Preview which integrates the Kubernetes RBAC authorization with Azure RBAC [1].
+This means you can define Kubernetes roles in the same system you define roles for Azure resources, and AKS will use Azure to authorize Kubernetes access requests.
+
+1.  https://docs.microsoft.com/en-us/azure/aks/concepts-identity
+2.  https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#containers
+3.  https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+
+### Privileged Access Management
+
+As discussed, identities should be managed via Azure Active Directory.
+Azure provides guidance and AD functionality for managing privileged identities, such as the Privileged Access Management notification service.
+
+1.  https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/directory-admin-roles-secure
+
+## Encryption & secure data management
+
+### Encryption of data at rest
```

Comment:
Encryption of Data at Rest

We need to keep it consistent across the document.

Reply:
@TLATER - I have addressed in the following PR to your fork ...

- https://github.com/TLATER/cloud-service-certification/pull/2